app/views/users/create_user.erb

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <link rel="stylesheet" href="/css/normalize.css">
```

Comment:
these link tags and the entire html layout should only be within your `layout.erb`. it will be automatically rendered on each individual `erb` file throughout the scope of your application.